### PR TITLE
Add blog post

### PR DESCRIPTION
**Pull request by flosse**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ Contributions welcome. Add links through pull requests or create an issue to sta
 
 ## Books
 - [Engineering Rust Web Applications](https://erwabook.com/) - Diesel, Rocket and Seed.
+- Porting a JavaScript App to WebAssembly with Rust [Part 1](https://slowtec.de/posts/2019-12-20-porting-javascript-to-rust-part-1.html)
```
Comment (MartinKavik) [inline README.md]: It should follow awesome-list conventions - i.e. it should be in format: `- [aaa](bbb) - Ccc.`
E.g. 
- [Porting a JS app to Rust](https://slowtec.de/posts/2019-12-20-porting-javascript-to-rust-part-1.html) - Porting a JavaScript App to WebAssembly with Rust (blog series).

**Comment by MartinKavik** [comment]
Very nice! 
Sorry for the delay.

Feedback: 
1.  Typo `recusively` (below `Step 4: Create modules`).
2. Where can I subscribe to get next parts once they are published?
3. Hello world example:
    - `App` is in the `prelude` so you can remove `seed` from `seed::App::builder(up..`.
    - You can replace `#[macro_use] extern crate seed;` with `use seed::{*, prelude::*};` (just tip).

File: README.md [hunk 2]
```diff
@@ -28,6 +28,8 @@ Contributions welcome. Add links through pull requests or create an issue to sta
 ## Books
 - [Engineering Rust Web Applications](https://erwabook.com/) - Diesel, Rocket and Seed.
 
+- [Porting a JS app to Rust](https://slowtec.de/posts/2019-12-20-porting-javascript-to-rust-part-1.html) - Porting a JavaScript App to WebAssembly with Rust (blog series)
```
Comment (MartinKavik) [inline README.md]: Missing `.` add the end

**Comment by flosse** [comment]
>     1. Typo `recusively` (below `Step 4: Create modules`).

fixed :)

>     2. Where can I subscribe to get next parts once they are published?

https://slowtec.de/feed.rss

>     3. Hello world example:
>        
>        * `App` is in the `prelude` so you can remove `seed` from `seed::App::builder(up..`.
>        * You can replace `#[macro_use] extern crate seed;` with `use seed::{*, prelude::*};` (just tip).

I used v0.5.0 and expected the changes in v0.6.0 ;-)

**Comment by MartinKavik** [comment]
I don't know how to use rss feed :D

**Comment by MartinKavik** [comment]
Do you want to copy/paste the article to medium or dev.to, so people can follow you?

**Pull request merged**